backend/app/agents/devrel/github/tools/github_support.py

```
@@ -0,0 +1,115 @@
+                "status": "error",
+                "message": f"Failed to fetch repositories for {org}",
+                "details": repos,
+            }
+
+        return {
+            "status": "success",
+            "message": f"Repositories for {org}",
+            "repositories": repos,
+        }
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)